### DISCUSSION TITLE
> Cannot destructure property 'HTTP2_HEADER_CONTENT_ENCODING' of 'http2.constants' as it is undefined.

### DISCUSSION THREAD

**Discussion by vickygarud** [cypress-io/cypress#27447] [Questions and Help]
Trying to feth  gmail on cypress spec file using this library 
` npm install @google-cloud/local-auth`
able to fetch mail details on console but not able to fetch mail on cypress test runner getting below error

<img width="686" alt="image" src="https://github.com/cypress-io/cypress/assets/104687841/2286dee2-5c7b-4fdf-88e6-9df5499ed175">

**Comment by luacsimoes** [comment]
Did you find out?
